### PR TITLE
Fix/terraform apply changes

### PR DESCRIPTION
**Pull request by elisehuard**
no battle plan survives contact with the enemy: all changes required to make the new subnet bits work well.

### CODE REVIEW

File: src/terraboot/infra.clj [hunk 1]
```diff
@@ -1,10 +1,16 @@
...
 
 (def infra-path "infra/")
 
+(defn get-config
+  "Gets info from a config file."
+  [url]
+  (edn/read-string (slurp url)))
+
 (defn generate-json [target]
   (let [account-number "165664414043"
```
Comment (jasebell) [inline src/terraboot/infra.clj:15]: Just a small thought, it may be prudent to find a way to externalise the account number from a public repo.

Comment (elisehuard) [inline src/terraboot/infra.clj:15]: That's the thing, I would advocate for the infra.clj part to be in a private repo.  terraboot lib public, all specific data private

Comment (jasebell) [inline src/terraboot/infra.clj:15]: A good enough reason.

**Comment by jasebell** [comment]
:shipit:

**Pull request merged**